design-toolkit/components/src/components/notice/styles.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { tv } from '@/lib/utils';
+
+export const NoticeStyles = tv({
+  slots: {
+    notice: [
+      'group/notice fg-primary-bold flex rounded-medium text-body-s shadow-elevation-overlay outline',
+      'size-small-min-h-[54px] size-small:min-w-[120px] size-small:max-w-[320px] size-small:flex-col size-small:gap-m size-small:p-m',
+      'size-medium:min-h-[64px] size-medium:min-w-[200px] size-medium:max-w-[640px] size-medium:gap-l size-medium:p-l',
+      'color-info:bg-info-muted color-info:outline-info-bold',
+      'color-advisory:bg-advisory-muted color-advisory:outline-accent-primary-bold',
+      'color-normal:bg-normal-muted color-normal:outline-normal-bold',
+      'color-serious:bg-serious-muted color-serious:outline-serious-bold',
+      'color-critical:bg-critical-muted color-critical:outline-critical-bold',
+    ],
+    content: [
+      'flex flex-1 items-center group-size-medium/notice:gap-l group-size-small/notice:gap-m',
+    ],
+    region: [
+      'absolute flex flex-col gap-y-s',
```

Comment:
The region should only be absolute if placement has been set. By default there shouldn't be a placement and the list should render in place